### ISSUE TITLE
Welcome tour: English localization missing for some strings under "Trainings" tab

### ISSUE THREAD

**Issue by wdconinc** [sleede/fab-manager#226]
When setting up a new production server following the production_readme.md on August 9, 2020, and logging in for the first time as the admin user, the tour includes French sentences even in the English localization settings.

# Expected behavior
Only english text should show up for new users.

# Actual behavior
Second paragraph under "Trainings" is in French:
![image](https://user-images.githubusercontent.com/4656391/89741931-02ad1c80-da5b-11ea-8e97-56a6e99529a8.png)

# Additional information
Relevant section of `config/env`:
```
RAILS_LOCALE=en
APP_LOCALE=en
MOMENT_LOCALE=en
SUMMERNOTE_LOCALE=en-US
ANGULAR_LOCALE=en-ca    
FULLCALENDAR_LOCALE=en-ca

POSTGRESQL_LANGUAGE_ANALYZER=english
```

**Comment by wdconinc** [comment]
Related. Typo: "EExplain" should be "Explain" under Customization > Privacy Policy:

![image](https://user-images.githubusercontent.com/4656391/89742026-dd6cde00-da5b-11ea-9f0a-aa16e4a878e4.png)

**Comment by wdconinc** [comment]
For the translation of "Certaines formations peuvent être un préalable à la réservation de certaines machines" I'd suggest "Trainings can be set as prerequisites before allowing reservation of certain machines."

**Comment by sylvainbx** [comment]
Hi @wdconinc ,
Thanks for your report, I'll check that asap.

**Comment by sylvainbx** [comment]
Fixed with [release 4.5.5](https://github.com/sleede/fab-manager/releases/tag/v4.5.5)
